src/reducers/bit-eraser-size.js

```diff
@@ -0,0 +1,31 @@
+import log from '../log/log';
+
+const CHANGE_BIT_ERASER_SIZE = 'scratch-paint/eraser-mode/CHANGE_BIT_ERASER_SIZE';
+const initialState = 20;
```

Comment:
I noticed that this is different from the eraser size in vector mode. Is that on purpose?

Reply:
No, that's a mistake